build_scripts/windows/azure-cli.wixproj

```diff
@@ -8,6 +8,8 @@
     <WixTasksPath>wixtasks.dll</WixTasksPath>
     <AzureCliSource>artifacts\cli</AzureCliSource>
     <LinkerAdditionalOptions>-fv</LinkerAdditionalOptions>
+    <HarvestDirectorySuppressAllWarnings>5150;5151</HarvestDirectorySuppressAllWarnings>
```

Comment:
I saw from https://wixtoolset.org/documentation/manual/v3/msbuild/target_reference/harvestdirectory.html that `HarvestDirectorySuppressAllWarnings` should be a boolean:

![image](https://user-images.githubusercontent.com/4003950/183560557-e2adbe96-e543-4602-8195-d59952b24fb1.png)

I am wondering if I misunderstand anything.

Reply:
No, my bad. I copied the wrong one (to avoid typos). Meant to copy the one with "...SpecificWarnings".